### ISSUE TITLE
Installing instructions could be better

### ISSUE THREAD

**Issue by MatejHantabal** [AsadAzam/DiscordX#1]
The steps on installing you provided are hard to follow and in my case, doing what you said did not work (moved to applications but no dialogue came up) - maybe an installation video would be nice to have.

**Comment by AsadAzam** [comment]
My Developer License has expired and I don't think I'd be renewing it for a while, you'd have to open the project on Xcode and install it manually.
Steps:
1. Download the project, and open it on Xcode.
2. Go to Signing & Capabilities, chose **Team** and Select **Sign to Run Locally** in Signing Certificate.
3. Build and Run the App.
4. (Optional) Go to Products under the Navigator and right click on DiscordX.app, click on Show in Finder. Copy/Cut - Paste in Applications folder and it should be easier to launch next time.

**Comment by MatejHantabal** [comment]
Thank you, this worked. Nice one :)
